zerver/openapi/zulip.yaml

```diff
@@ -7778,7 +7778,7 @@ paths:
         - name: desktop_icon_count_display
           in: query
           description: |
-            Unread count summary (appears in desktop sidebar and browser tab)
+            Unread count badge (appears in desktop sidebar and browser tab)
```

Comment:
I think we can update these descriptions to be more descriptive like we do for other settings. Can be done in a separate PR as well, just mentioning it here for feedback.

Reply:
That seems worth doing, but I think is appropriate to be a separate PR.